### PR TITLE
FIX: Updating french translation

### PR DESCRIPTION
**Pull request by Omega974**
FIX: Updating french translation
When merged this pull request will:

Update the translation
https://github.com/Vdauphin/HeartsAndMinds/issues/400

**Final test:**
- [x] local
- [x] server

**Screenshots**

### CODE REVIEW

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 1]
```diff
@@ -760,18 +763,23 @@
             </Key>
             <Key ID="STR_BTC_HAM_RESP_TICKETPRISONER">
                 <Original>Attribute tickets from prisoners to (Enable only if tickets are not share):</Original>
+                <French>Nombre de ticket que rapporte un prissonier (Activé cette option seulement si les tickets ne sont pas partagés): </French>
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml:766]:
```suggestion
                <French>Nombre de ticket que rapporte un prisonier (Activé cette option seulement si les tickets ne sont pas partagés): </French>
```

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 2]
```diff
@@ -760,18 +763,23 @@
...
             </Key>
             <Key ID="STR_BTC_HAM_RESP_ALLCONNECTED">
                 <Original>All connected players</Original>
+                <French>Tous les joueurs sont connectés</French>
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml:770]:
```suggestion
                <French>A tous les joueurs connectés</French>
```

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 3]
```diff
@@ -760,18 +763,23 @@
...
             </Key>
             <Key ID="STR_BTC_HAM_RESP_PLAYER">
                 <Original>To the player interacting with the logistic point</Original>
+                <French>Le joueur interagissant avec le point logistique</French>
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml:774]:
```suggestion
                <French>Au joueur interagissant avec le point logistique</French>
```

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 4]
```diff
@@ -760,18 +763,23 @@
...
             </Key>
             <Key ID="STR_BTC_HAM_RESP_DEADPLAYER">
                 <Original>To connected player with 0 ticket left</Original>
+                <French>Le joueur connecté à un solde de 0 ticket</French>
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml:778]:
```suggestion
                <French>Au joueur connecté avec un solde de 0 ticket</French>
```

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 5]
```diff
@@ -760,18 +763,23 @@
...
             </Key>
             <Key ID="STR_BTC_HAM_RESP_LOWESTPLAYER">
                 <Original>To connected player with the lowest ticket number (0 included)</Original>
+                <French>Le joueur avec le plus faible solde de ticket (0 inclus)</French>
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml:782]:
```suggestion
                <French>Au joueur avec le plus faible solde de ticket (0 inclus)</French>
```

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 6]
```diff
@@ -796,6 +804,7 @@
             </Key>
             <Key ID="STR_BTC_HAM_PARAM_SLOT_ISSHARE">
                 <Original>Each slot is share between players</Original>
+                <French>Les slot sont partagés entre tous les joueurs</French>
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml:807]:
```suggestion
                <French>Les slots sont partagés entre tous les joueurs</French>
```

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 7]
```diff
@@ -820,31 +829,31 @@
                 <German>Verwackeln beim Zielen</German>
                 <Portuguese>Habilidade de Pontaria</Portuguese>
                 <Chinesesimp>瞄准抖动(AimingShake)</Chinesesimp>
-                <French>TremblementDeVisée</French>
+                <French>Tremblement De Visée</French>
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml:832]:
```suggestion
                <French>Tremblement de visée</French>
```

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 8]
```diff
@@ -820,31 +829,31 @@
...
                 <Czech>Třes zaměřování</Czech>
             </Key>
             <Key ID="STR_BTC_HAM_PARAM_SKILL_AIMSPEED">
                 <Original>AimingSpeed</Original>
                 <German>Zielgeschwindigkeit</German>
                 <Portuguese>Velocidade de Pontaria</Portuguese>
                 <Chinesesimp>瞄准速度(AimingSpeed)</Chinesesimp>
-                <French>VitesseDeVisée</French>
+                <French>Vitesse De Visée</French>
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml:840]:
```suggestion
                <French>Vitesse de visée</French>
```

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 9]
```diff
@@ -820,31 +829,31 @@
...
                 <Czech>Rychlost zaměřování</Czech>
             </Key>
             <Key ID="STR_BTC_HAM_PARAM_SKILL_SPOTDIST">
                 <Original>SpotDistance</Original>
                 <German>Aufklärungsentfernung</German>
                 <Portuguese>Distância de Reconhecimento</Portuguese>
                 <Chinesesimp>目标识别距离(SpotDistance)</Chinesesimp>
-                <French>DistanceDeRepérage</French>
+                <French>Distance De Repérage</French>
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml:848]:
```suggestion
                <French>Distance de repérage</French>
```

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 10]
```diff
@@ -820,31 +829,31 @@
...
                 <Czech>SpotDistance</Czech>
             </Key>
             <Key ID="STR_BTC_HAM_PARAM_SKILL_SPOTTIME">
                 <Original>SpotTime</Original>
                 <German>Aufklärungsgeschwindigkeit</German>
                 <Portuguese>Velocidade de Reconhecimento</Portuguese>
                 <Chinesesimp>目标识别时间(SpotTime)</Chinesesimp>
-                <French>TempsDétection</French>
+                <French>Temps De Détection</French>
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml:856]:
```suggestion
                <French>Temps de détection</French>
```

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 11]
```diff
@@ -860,7 +869,7 @@
                 <German>Nachladegeschwindigkeit</German>
                 <Portuguese>Velocidade de Recarga</Portuguese>
                 <Chinesesimp>装填时间(ReloadSpeed)</Chinesesimp>
-                <French>VitesseDeRechargement</French>
+                <French>Vitesse De Rechargement</French>
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml:872]:
```suggestion
                <French>Vitesse de rechargement</French>
```

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 12]
```diff
@@ -1078,6 +1087,7 @@
             </Key>
             <Key ID="STR_BTC_HAM_PARAM_OTHER_FLAGVEHLEADER">
                 <Original>On vehicle and team leader</Original>
+                <French>Sur les véhicule et le team leader</French>
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml:1090]:
```suggestion
                <French>Sur les véhicules et le team leader</French>
```

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 13]
```diff
@@ -2156,19 +2166,22 @@
             </Key>
             <Key ID="STR_BTC_HAM_ACTION_COPYPASTE_COPYI">
                 <Original>Copy inventory</Original>
+                <French>Copié l'enventaire</French>
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml:2169]:
```suggestion
                <French>Copier l'enventaire</French>
```

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 14]
```diff
@@ -2156,19 +2166,22 @@
...
             </Key>
             <Key ID="STR_BTC_HAM_ACTION_COPYPASTE_PASTEI">
                 <Original>Paste inventory</Original>
+                <French>Collé l'inventaire</French>
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml:2173]:
```suggestion
                <French>Coller l'inventaire</French>
```

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 15]
```diff
@@ -2156,19 +2166,22 @@
...
             </Key>
             <Key ID="STR_BTC_HAM_ACTION_RESTOREI">
                 <Original>Restore inventory</Original>
+                <French>Restoré l'inventaire</French>
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml:2177]:
```suggestion
                <French>Restorer l'inventaire</French>
```

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 16]
```diff
@@ -2156,19 +2166,22 @@
...
             </Key>
         </Container>
         <Container name="Action(ACE): Door">
             <Key ID="STR_BTC_HAM_ACTION_DOOR_BREAK">
                 <Original>Break door lock</Original>
                 <German>Brich die Tür auf</German>
-                <French>Casser la serrure</French>
+                <French>Forcer la porte</French>
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml:2184]:
```suggestion
                <French>Forcer la serrure</French>
```

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 17]
```diff
@@ -2785,6 +2798,7 @@
             </Key>
             <Key ID="STR_BTC_HAM_O_COMMON_NOPLAYERFTICKET">
                 <Original>None player need ticket</Original>
+                <French>Aucun joueur n'as besoin de ticket</French>
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml:2801]:
```suggestion
                <French>Aucun joueur n'a besoin de ticket</French>
```

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 18]
```diff
@@ -2846,9 +2860,11 @@
             </Key>
             <Key ID="STR_BTC_HAM_O_PASTE_NOCOPIEDI">
                 <Original>No copied inventory!</Original>
+                <French>L'inventaire n'as pas été copier</French>
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml:2863]:
```suggestion
                <French>L'inventaire n'a pas été copié</French>
```

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 19]
```diff
@@ -2846,9 +2860,11 @@
...
             </Key>
             <Key ID="STR_BTC_HAM_O_COPY_NOOBJECTS">
                 <Original>No objects around!</Original>
+                <French>Pas d'objet dans les environ!</French>
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml:2867]:
```suggestion
                <French>Pas d'objet dans les environs !</French>
```

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 20]
```diff
@@ -3107,6 +3123,7 @@
         <Container name="Logistic: Rearm">
             <Key ID="STR_BTC_HAM_LOG_REARM_COMPLETED">
                 <Original>Rearm completed.</Original>
+                <French>Réarmement éffectué.</French>
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml:3126]:
```suggestion
                <French>Réarmement effectué.</French>
```

**Review by Vdauphin — CHANGES_REQUESTED** [review]
Thank you very much for the help, some stuff still need to be fix

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml (hunk 13 above)
Comment (Omega974) [reply =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml]: "inventaire"

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 21]
```diff
@@ -667,7 +669,7 @@
                 <Original>Density of intel in house:</Original>
                 <Chinesesimp>房屋内的情报密度:</Chinesesimp>
                 <Czech>Hustota intelu v domě:</Czech>
-                <French>Densité du renseignements dans les maisons: </French>
+                <French>Densité du renseignements dans les maisons:</French>
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml:672]:
```suggestion
                <French>Densité du renseignements dans les maisons :</French>
```

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 22]
```diff
@@ -743,6 +745,7 @@
             </Key>
             <Key ID="STR_BTC_HAM_RESP_TICKETSLOST">
                 <Original>Number of tickets lost per death:</Original>
+                <French>Nombre de ticket perdu par mort:</French>
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml:748]:
```suggestion
                <French>Nombre de ticket perdu par mort :</French>
```

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 23]
```diff
@@ -760,18 +763,23 @@
             </Key>
             <Key ID="STR_BTC_HAM_RESP_TICKETPRISONER">
                 <Original>Attribute tickets from prisoners to (Enable only if tickets are not share):</Original>
+                <French>Nombre de ticket que rapporte un prisonier (Activer cette option seulement si les tickets ne sont pas partagés): </French>
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml:766]:
```suggestion
                <French>Nombre de ticket que rapporte un prisonier (Activer cette option seulement si les tickets ne sont pas partagés) : </French>
```

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 24]
```diff
@@ -2156,19 +2166,22 @@
             </Key>
             <Key ID="STR_BTC_HAM_ACTION_COPYPASTE_COPYI">
                 <Original>Copy inventory</Original>
+                <French>Copier l'enventaire</French>
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml:2169]:
```suggestion
                <French>Copier l'inventaire</French>
```

**Review by Vdauphin — CHANGES_REQUESTED** [review]
Thanks again!
Still some change

**Review by Vdauphin — APPROVED** [review]
Perfect!

**Comment by Omega974** [comment]
> Perfect!

Like you !

**Pull request merged**